urgent-care/src/test/java/gov/va/api/health/urgentcare/service/queenelizabeth/client/RestQueenElizabethClientTests.java

```diff
@@ -0,0 +1,94 @@
+package gov.va.api.health.urgentcare.service.queenelizabeth.client;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import gov.va.med.esr.webservices.jaxws.schemas.GetEESummaryResponse;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+import org.springframework.web.client.RestTemplate;
+
+public class RestQueenElizabethClientTests {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+
+  @Mock RestTemplate mockRestTemplate;
+
+  private RestQueenElizabethClient qeClient;
+
+  @Before
+  public void _init() {
+    MockitoAnnotations.initMocks(this);
+    qeClient =
+        new RestQueenElizabethClient("https://ee.va.gov:9334/getEESummary/", mockRestTemplate);
```

Comment:
Do we want this to be the real thing for a test?  In a mock?  Maybe remove real IP and port.

Reply:
Look again. Thats not the real thing.